jinja2/filters.py

```diff
@@ -940,6 +940,29 @@ def do_attr(environment, obj, name):
     return environment.undefined(obj=obj, name=name)
 
 
+@contextfilter
+def do_flip(context, value, name, *args, **kwargs):
+    """Applies a filter with arguments flipped in reverse order.
+    This becomes useful with filters like map where you can't easily re-order
+    the arguments yourself.
+
+    Say you want to find all numbers in a list that divides 10 you'd do it like
+    this_
+
+    .. sourcecode:: jinja
+
+        {{ [1,2,3,4,5,6,7,8,9] | select('flip', 'divisibleby', 10) | list }}
+
+
+    This would give you back `[1, 2, 4, 5]`.
+
+    .. versionadded:: 2.11
+    """
+    args = list(args)[::-1] + [value]
```

Comment:
`args = args[::-1] + (value,)`?

Reply:
I don't understand. :)

Reply:
avoids converting to a list - you don't need the mutability so no reason to not stay with tuples

Reply:
Ah! I didn't know `*args` was a tuple.

Reply:
Fixed!